pkgs/development/interpreters/python/graalpython/default.nix

```diff
@@ -1,21 +0,0 @@
-{ pkgs
-, lib
-, graalvm8
-, passthruFun
-, packageOverrides ? (self: super: {})
-, self
-}:
-
-let
-  passthru = passthruFun {
-    inherit self packageOverrides;
-    implementation = "graal";
-    sourceVersion = graalvm8.version;
```

Comment:
Even if support for Python hasn't been removed, this would fail anyway since `graalvm8` was removed quite a long time ago.

Reply:
Before:

```
nix-repl> legacyPackages.x86_64-linux.pythonInterpreters.graalpython37
error: graalvm8-ce has been removed by upstream
```